seam/seam.py

```diff
@@ -89,6 +90,31 @@ def __init__(
 
         Routes.__init__(self, client=self.client, defaults=self.defaults)
 
+    def create_paginator(
+        self, request: Callable, params: Optional[Dict[str, Any]] = None
```

Comment:
https://peps.python.org/pep-0570/#syntax-and-semantics
```suggestion
        self, /, *, request: Callable, params: Optional[Dict[str, Any]] = None
```

Reply:
What's the reason for making `self` positional-only if it's passed automatically?

Reply:
Also making `request` and `params` keyword-only will make the syntax you proposed [here](https://github.com/seamapi/python/pull/311#discussion_r2045279421) invalid

Reply:
Ah, I meant to suggest 
```suggestion
        self, request: Callable, params: Optional[Dict[str, Any]] = None, /, *)
```

Reply:
I don't think the asterisk has any effect here, since it only makes parameters that come after it keyword-only.